export const signUp = async (req, res) => {
    try {
        
    } catch (err) {

    }
};